packages/@tinacms/cli/src/cmds/start-server/server.ts

```diff
@@ -61,6 +61,39 @@ const gqlServer = async () => {
     return res.json(result)
   })
 
+  app.get('/list-branches', async (req, res) => {
+    try {
+      const { query } = req
+      const { owner, repo } = query
+      const result = await gqlPackage.listBranches({
+        auth: process.env.GITHUB_PERSONAL_ACCESS_TOKEN,
+        owner,
+        repo
+      })
+  
+      return res.json(result.data)
+    } catch(error) {
+      console.error('There was a problem fetching the branches.', error)
+    }
+  })
+
+  app.post('/create-branch', async (req, res) => {
+    try {
+      const { query } = req
+      const { owner, repo, name, branch } = query
```

Comment:
agree that we should rename the `branch` parameter because it's a little ambiguous here. `baseBranch` sounds good to me.